packages/framework/src/Model/Payment/Payment.php

```diff
@@ -119,13 +120,30 @@ public function addTransport(Transport $transport)
     public function setTransports(array $transports)
```

Comment:
If I have payment `$pay` and transport `$trans` linked together and call `$pay->setTransports([])` I should remove the link. Transport `$trans` will still have the link to the payment `$paym`.

Reply:
Fixed